client/me/purchases/purchase-item/index.jsx

```diff
@@ -100,6 +102,37 @@ class PurchaseItem extends Component {
 			);
 		}
 
+		if ( isWithinIntroductoryOfferPeriod( purchase ) && isIntroductoryOfferFreeTrial( purchase ) ) {
+			if ( isRenewing( purchase ) ) {
+				return translate( 'Free trial ends on {{span}}%(date)s{{/span}}, renews automatically', {
```

Comment:
To be a little more in line with the current renewal status, can we change this to `Free trial ends on {{span}}%(date)s{{/span}}, renews automatically at %(amount)s`?